src/table/TableWrapper.jsx

```diff
@@ -48,6 +48,7 @@ export default function TableWrapper(props) {
 
   const handleChangePage = (event, newPage) => {
     setPageInfo({ top: newPage * rowsPerPage, height: rowsPerPage });
+    setfocusedCellCoord([0, 0]);
```

Comment:
why is this added?

Reply:
I guess it's not required any more since we do it in `handleResetFocus`

Reply:
✅ I'll remove it